image.yaml

```diff
@@ -128,8 +130,10 @@ modules:
             version: '1.0'
 packages:
       repositories:
-          - jboss-os
-          - jboss-rhscl
+          - name: jboss-os
```

Comment:
Should be possible to completely remove these repos if the corresponding content sets are defined instead. Per [this CEKit note](https://docs.cekit.io/en/latest/guidelines/repositories.html#content-sets) other repos are ignore if content sets are defined. So we could drop them altogether and define proper content sets.